README.md

```diff
@@ -1,5 +1,145 @@
 
-# Specification 'origin-policy'
+# Explainer: Origin Manifest
 
-This is the repository for origin-policy. You're welcome to contribute! Let's make the Web rock our socks
-off!
+**tl;dr** Origin Manifest (formerly Origin Policy) is a web platform mechanism
+that aims to shift configuring the origin from web applications to the origin
+itself. For this the origin provides a manifest file in a predefined well-known
```

Comment:
"from web applications to the origin"

Reply:
I think it would be helpful to make it explicit that you're moving configuration options from response-bound headers to an out-of-band declaration that can be safely assumed to represent The Origin Itself<sup>&trade;</sup>.